config/schedule.yml

```diff
@@ -78,6 +78,14 @@ award_contributor_badges_from_github:
     - ""
     - award_contributor_from_github
     - ""
+award_community_wellness_badges:
+  description: "Awards 'Community Wellness' badges to users (runs daily at 00:45 UTC)"
+  cron: "45 0 * * *"
```

Comment:
Some other jobs run at ~00:00 UTC so I chose minute 45 to reduce the load on that hour

Reply:
It runs in less than 2 seconds. We checked the original query (this one has been slightly tweaked since) with @jgaskins and @jeremyf and looked good for a _"once or twice a day cadence"_ since it was doing well on index use despite having few unique values on `reaction.category`, if I remember correctly 😄 

1. Here is the [up-to-date Blazer query](https://dev.to/admin/blazer/queries/683)
2. Here is the [up-to-date EXPLAIN ANALYZE of the Blazer query](https://dev.to/admin/blazer/queries/684)
   - Which I can't myself read too well, but I think having `loops=1` throughout may be a good thing (?) 😅 

Note: Using these links for the docs